core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2980,6 +3094,24 @@ private void setAttributeFromEntry(SetAttributeEntry entry)
     return mBlockMaster.getWorkerInfoList();
   }
 
+  private void checkAndAppendAuditLog(String command, String srcPath, String dstPath, AuditContext context) throws AccessControlException {
+    if (!mAsyncAuditLogWriter.isEnabled()) { return; }
+    MasterAuditContext masterAuditContext = (MasterAuditContext) context;
```

Comment:
To be more type safe, why not just take `context` as `MasterAuditContext` to avoid conversion?